packages/debugger/src/panels/breakpoints/body.tsx

```diff
@@ -73,6 +83,8 @@ const BreakpointsComponent = ({
           key={entry[0]}
           breakpoints={entry[1]}
           model={model}
+          selectedLine={selectedBreakpoint?.line}
+          selectedPath={selectedBreakpoint?.source?.path}
```

Comment:
Can you pass the `selectedBreakpoint` instead? Here is my reasoning:
- at some point we may decide to support multiple breakpoints per line and then we also want to add `column`
- maybe someone wants to support ranges and then we need to add `endLine` and `endColumn`
- so now we redefined half of the `Breakpoint` interface but we could have just passed it around all the time
- passing `line` and `path` would be better if re-rendering was a concern, but here the breakpoint is passed within `breakpoints` prop anyways so rerendering would happen regardless (if the breakpoint object was redefined).
- also noting that sometimes `Breakpoint` will have `id` property which can be used as a faster check (over comparing the relevant properties).

```suggestion
          selectedBreakpoint={selectedBreakpoint}
```